src/Microsoft.Health.Dicom.Api/Modules/SecurityModule.cs

```diff
@@ -64,5 +70,13 @@ public void Load(IServiceCollection services)
 
             services.AddSingleton<IClaimsExtractor, PrincipalClaimsExtractor>();
         }
+
+        internal IEnumerable<string> GetAuthenticationAudiences()
+        {
+            return _securityConfiguration.Authentication.Audiences ?? new[]
+            {
+                _securityConfiguration.Authentication.Audience,
```

Comment:
Do you get a reasonable error message when one of the entries is empty, like `Audience` being null or ""?

Reply:
In the current implementation, it actually changes the behavior slightly if you don't have either being set. 

I'm going to update the code to maintain the current behavior if neither is specified.

Reply:
Also, you get a message like `Microsoft.IdentityModel.Tokens.SecurityTokenInvalidAudienceException: IDX10214: Audience validation failed. Audiences: 'https://example.com'. Did not match: validationParameters.ValidAudience: 'null' or validationParameters.ValidAudiences: 'empty'.`